api/server.js

```diff
@@ -30,18 +30,13 @@ app.post('/jwt_auth', process.env.GOOGLE_AUTH_MOCK ? gAuthMock : gAuth)
 app.use(pgFetch)
 app.use(pgJwt(JWT_SECRET))
 
-// graphql endpoint
-app.use('/graphql', postgraphql(
-    `postgres://${env.PGUSER}:${env.PGPASSWORD}@${env.PGHOST}:${env.PGPORT}/${env.PGDATABASE}`,
-    'give_me_time_public',
-    {
-        development: true,
-        secret: JWT_SECRET,
-        log: true,
-        // any non logged user is not admin
-        anonymousRole: 'give_me_time_user'
-    }
-))
+// endpoints
+app.post('/project', handlers.project.index.post)
+app.post('/project/give/:id', handlers.project.give.index.post)
+app.get('/project/:id', handlers.project.index.get)
+app.delete('/project/:id', handlers.project.index.delete)
+app.get('/projects', handlers.projects.index.get)
+app.post('/login', handlers.login.index.post)
 
 console.log('Listening to port 3000')
-app.listen(3000)
\ No newline at end of file
+app.listen(3000)
```

Comment:
Not related to this file but I could not comment on app/bundle.js for some reason.
app/bundle.js should not be committed, it's huge.